docs/_docs/dev-guide/imix.md

```diff
@@ -1,212 +1,285 @@
----
```

Comment:
This is spooky just double check that you're not accidentally deleting upstream docs on accident.